frontend/src/helpers/base-url.ts

```diff
@@ -0,0 +1,7 @@
+const baseUrl = (): string =>
+    window.location.protocol +
+    '//' +
+    window.location.hostname +
+    (window.location.port ? ':' + window.location.port : '');
+
+export { baseUrl };
```

Comment:
I'm sure it can be refactored for a better readability

Reply:
Done